docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -372,35 +372,36 @@ As Timefold Solver will mutate the <<planningVariable,genuine>> or <<shadowVaria
 As Java's `enum` and `record` types are immutable by design, these can not be used as planning entities.
 
 
-[#planningEntityDifficulty]
-=== Planning entity difficulty
+[#planningEntitySorting]
+=== [[planningEntityDifficulty]]Planning entity sorting
```

Comment:
I've never seen this syntax before. Have you tried both anchors in the deployed documentation? Does it work?